packages/contracts/contracts/interfaces/v0.8.x/IPMPV0.sol

```diff
@@ -52,11 +52,13 @@ interface IPMPV0 is IWeb3Call {
      * @param coreContract The address of the core contract.
      * @param tokenId The token ID for which parameters were configured.
      * @param pmpInputs Array of parameter inputs that were configured.
+     * @param authAddresses Array of addresses that authenticated the parameters. Aligned by index with pmpInputs.
      */
     event TokenParamsConfigured(
         address coreContract,
         uint256 tokenId,
-        PMPInput[] pmpInputs
+        PMPInput[] pmpInputs,
```

Comment:
What do you think about adding a new field on the PMPInput struct, `configuringAddress`, instead of isolating authAddresses as a new parameter?

Reply:
I chose this format because the `pmpInputs` is the input struct to the configure token params function. Totally agree that it would be ideal to add the new field to the PMPInput struct from the event's perspective, but it's also used as function input so we would need a new struct for the event. I didn't want to spend the gas during execution to copy all the calldata a different struct in memory, so figured best to have a slightly awkward event param for efficiency. Happy to chat more about it though!

Reply:
ah makes sense! 🙏  appreciate the details